namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/reporting-landing-access.tf

```diff
@@ -57,5 +59,6 @@ resource "kubernetes_secret" "reporting_aws_secret" {
     user_arn           = aws_iam_user.reporting_user.arn
     access_key_id      = aws_iam_access_key.reporting_user.id
     secret_access_key  = aws_iam_access_key.reporting_user.secret
+    bucket_name        = "eu-west-2-delius-mis-dev-dfi-extracts"
```

Comment:
given the current `destination_bucket = "s3://eu-west-2-delius-mis-dev-dfi-extracts/dfinterventions/dfi"` value I think we'd also need a `bucket_prefix = "dfinterventions/dfi/"`